src/lesson6/Circle.java

```diff
@@ -0,0 +1,13 @@
+package lesson6;
+
+public class Circle {
+    private final int RADIUS;
+
+    public Circle(int radius) {
+        this.RADIUS = radius;
+    }
+
+    public double getArea(){
+        return Math.PI * Math.pow(this.RADIUS, 2);
+    }
```

Comment:
і тут можна без this, так як в даному методі доступний тільки один radius
```suggestion
        return Math.PI * Math.pow(radius, 2);
    }
```